src/main/java/edu/kit/compiler/lexer/StringTable.java

```diff
@@ -33,7 +33,7 @@ public String retrieve(int hash) {
         if(hash < size() && hash >= 0) {
             return strings.get(hash);
         }
-        return null;
+        throw new IllegalArgumentException("No string present for this value.");
```

Comment:
Valid behavior, but please change the doc accordingly. Also we should probably add a `contains` method to safely access strings.